src/logic/currencyValues/__tests__/fetchSafeTokens.test.ts

```diff
@@ -1,7 +1,10 @@
-import axios from 'axios'
+import axios, { AxiosResponse } from 'axios'
```

Comment:
is AxiosResponse being used anywhere?

Reply:
The function tested here was returning an AxiosResponse type, but it's not necessary and I added the changes to directly return the info